packages/suite/src/components/suite/Preloader/__tests__/Preloader.test.tsx

```diff
@@ -218,7 +226,7 @@ describe(`${Preloader.name} component`, () => {
         const { unmount } = renderWithProviders(store, <Index app={store.getState().router.app} />);
 
         expect(findByTestId('@connect-device-prompt')).not.toBeNull();
-        fireEvent.click(findByTestId('@onboarding/expand-troubleshooting-tips/toggle'));
+        fireEvent.click(findByTestId('@onboarding/troubleshooting-tips/button'));
```

Comment:
IMHO it's really bad practice to check hidden content, it was pretty hard to debug 🤷‍♂️